src/meta/src/hummock/compaction/compaction_config.rs

```diff
@@ -49,7 +49,7 @@ impl CompactionConfigBuilder {
                 compression_algorithm: vec![
                     "None".to_string(),
                     "None".to_string(),
-                    "Lz4".to_string(),
+                    "None".to_string(),
```

Comment:
Because I found that if we compress data since level 2. the data will be block in level 1 and it will cause write-amplification larger and larger. Because the slower compaction run, the data wait in level 1 will be larger. The larger level-1 is, the compaction runs from level-0 to level-1 would be slower.